bin/node-pg-migrate.ts

```diff
@@ -528,9 +528,12 @@ if (action === 'create') {
   if (!DB_CONNECTION) {
     const cp = new ConnectionParameters();
 
-    if (!cp.host && !cp.port && !cp.database) {
+    if (
+      !process.env[argv[databaseUrlVarArg]] &&
+      (!process.env.PGHOST || !cp.user || !cp.database)
```

Comment:
Hi, I found an issue with this logic. When we use a config file, it won't bother it, and the config file becomes useless. Is it intentional that the config won't be used?